contracts/metagov/VeBalDelegatorPCVDeposit.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "./SnapshotDelegatorPCVDeposit.sol";
+import "./utils/VoteEscrowTokenManager.sol";
+import "./utils/LiquidityGaugeManager.sol";
+
+/// @title 80-BAL-20-WETH BPT PCV Deposit
+/// @author Fei Protocol
+contract VeBalDelegatorPCVDeposit is
```

Comment:
should this include governor bravo compatibility? Vote escrow is a hard action to undo

Reply:
We can add OZGovernor capability (that have a contract ready & tested), but when I asked Kia he said that Balancer does everything with snapshot + multisigs, no governor

Reply:
Can you follow up and make sure they have no intentions of putting anything on-chain?

Reply:
including for example a nope-dao?